partner_identification/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```

Comment:
coding line is not necessary in v11 (P3)
remove copyright notice from `__init__.py` files too

Reply:
MQT is not ready yet and logs warnings... I was trying to get a clear build. I'll remove them anyway ;)

Reply:
But remember that warnings don't change green status.